src/commands/insert-temporary-identity.ts

```diff
@@ -19,8 +20,7 @@ async function insertTemporaryIdentity(
     return putUser(updatedUser);
   }
 
-  // TODO: better deal with this error
-  throw new Error("User not found");
+  throw new NoUserFound();
```

Comment:
```suggestion
  throw new NoUserFoundError();
```
😁

Reply:
I did it for this one, and added a task on clickup to add the suffix on the other ones.